website/www/site/content/en/documentation/sdks/yaml.md

```diff
@@ -708,6 +708,49 @@ options:
   streaming: true
 ```
 
+## Jinja Templatization
+
+It is a common to want to run a single Beam pipeline in different contexts
+and/or with different configurations.
+When running a YAML pipeline either locally or via the gcloud,
```

Comment:
nit: should we re-word to say `locally or against a remote pipeline runner` ? as gcloud seem restrictive.

Reply:
Yeah, I agree this isn't worded well. It's not really about the runner that's used, but the entrypoint. (E.g. we don't support this for embedded YamlTransform.)

Clarified that it's about both `apache_beam.yaml.main` (which an be local or various remote runners) and `gcloud` (which at the end of the day is `apache_beam.yaml.main`, but isn't s obvious and doesn't require beam to be installed).

Reply:
ty for the clarification!